swarm/swarm_at_scale/deploy-app.md

```diff
@@ -252,7 +252,7 @@ Now, you can test your application.
 ## Extra Credit: Deployment with Docker Compose
 
 Up to this point, you've deployed each application container individually. This
-can be cumbersome especially because their are several different containers and
+can be cumbersome especially because they are several different containers and
```

Comment:
I also wondered which one is better. Then, I chose "they are". Hard to tell the differences between them. Anyway, I change it to "there are".